### PR TITLE
Replace update notices with notifications

### PR DESCRIPTION
**Pull request by X1nto**

### CODE REVIEW

**Comment by Nuckyz** [comment]
rebase from main and make the notifications permanent using the `permanent: true` option

**Comment by X1nto** [comment]
Uhhh what the fuck happened with commits

File: src/Vencord.ts [hunk 1]
```diff
@@ -49,32 +49,30 @@ async function init() {
             if (Settings.autoUpdate) {
                 await update();
                 const needsFullRestart = await rebuild();
-                setTimeout(() => {
-                    showNotice(
-                        "Vencord has been updated!",
-                        "Restart",
-                        () => {
+                if (Settings.autoUpdateNotification)
+                    await showNotification({
```
Comment (Nuckyz) [inline src/Vencord.ts:53]:
```suggestion
                    showNotification({
```

File: src/Vencord.ts [hunk 2]
```diff
@@ -49,32 +49,30 @@ async function init() {
...
-                    showNotice(
-                        "A Vencord update is available!",
-                        "View Update",
-                        () => {
-                            popNotice();
-                            SettingsRouter.open("VencordUpdater");
-                        }
-                    );
-                }, 10_000);
+                await showNotification({
```
Comment (Nuckyz) [inline src/Vencord.ts:68]:
```suggestion
                showNotification({
```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

File: src/Vencord.ts (hunk 1 above)
Comment (X1nto) [reply src/Vencord.ts:53]: Any particular reason why awaiting is a bad thing here?

Comment (Nuckyz) [reply src/Vencord.ts:53]: Actually just cuz there is no reason to await, let the next code run as soon as possible

Comment (X1nto) [reply src/Vencord.ts:53]: sure then

File: src/components/VencordSettings/Updater.tsx
Comment (Nuckyz) [inline src/components/VencordSettings/Updater.tsx:1]: https://github.com/Vendicated/Vencord/pull/558/files#diff-aeca2fccccc4f3357e578dafdd5dd914e6841e9222065d3a5392822fe0624e3dR208

Should probably change to Shows a notification on startup

File: src/components/VencordSettings/Updater.tsx [hunk 3]
```diff
@@ -217,6 +217,14 @@ function Updater() {
             >
                 Automatically update
             </Switch>
+            <Switch
+                value={settings.autoUpdateNotification}
+                onChange={(v: boolean) => settings.autoUpdateNotification = v}
+                note="Shows a notification when Vencord automatically updates"
+                disabled={!settings.autoUpdate}
+            >
+                Notify auto-update completion
```
Comment (Nuckyz) [inline src/components/VencordSettings/Updater.tsx:226]:
```suggestion
                Get notified when an automatic update completes
```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

**Review by Nuckyz — APPROVED** [review]

**Pull request merged**